domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/service/CaseTypeEntityToCaseTypeLiteDTOMapperTest.java

```diff
@@ -0,0 +1,173 @@
+import uk.gov.hmcts.ccd.definition.store.repository.model.CaseTypeLite;
+import uk.gov.hmcts.ccd.definition.store.repository.model.Jurisdiction;
+import uk.gov.hmcts.ccd.definition.store.repository.model.StateLite;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Date;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.hamcrest.Matchers.hasItems;
+import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+class CaseTypeEntityToCaseTypeLiteDTOMapperTest {
+
+    private static final String LIVE_FROM = "2017-02-02";
+
+    private static final String LIVE_TO = "2018-03-03";
+
+    private CaseTypeEntityToCaseTypeLiteDTOMapper classUnderTest = new CaseTypeEntityToCaseTypeLiteDTOMapperImpl();
+
+    private CaseTypeEntityToCaseTypeLiteDTOMapper spyOnClassUnderTest;
+
+    @BeforeEach
+    public void setupSpy() throws Exception {
+        spyOnClassUnderTest = spy(classUnderTest);
+    }
+
+    private StateEntity stateEntity() {
+        StateEntity stateEntity = new StateEntity();
+
+        stateEntity.setName("Name");
+        stateEntity.setReference("Id");
+        return stateEntity;
+    }
+
+    @Nested
+    @DisplayName("Should return a StateLite whose fields match those in the StateEntity")
+    public class MapStateEntitySubsetTests {
+        @Test
+        public void testMapEmptyStateEntity() throws Exception {
+            StateEntity stateEntity= new StateEntity();
+
+            StateLite stateLite = classUnderTest.map(stateEntity);
+
+            // Assertions
+            assertNull(stateLite.getId());
+            assertNull(stateLite.getName());
+        }
+
+        @Test
+        public void testMapSubsetStateEntity() throws Exception {
+            StateEntity stateEntity = stateEntity();
+
+            StateLite stateLite = classUnderTest.map(stateEntity);
+
+            // Assertions
+            assertEquals(stateLite.getId(), stateEntity.getReference());
+            assertEquals(stateLite.getName(), stateEntity.getName());
+        }
+
+    }
+
+    @Nested
+    @DisplayName("Should return a CaseTypeLite whose fields match those in the CaseTypeEntity")
+    public class MapCaseTypeEntitySubsetTests {
+
+
+        @Test
+        public void testMapSubsetCaseTypeEntity() throws Exception {
+            CaseTypeEntity caseTypeEntity = caseTypeEntity();
+
+            CaseTypeLite caseTypeLite = classUnderTest.map(caseTypeEntity);
+
+            // Assertions
+            assertEquals(caseTypeLite.getId(), caseTypeEntity.getReference());
+            assertEquals(caseTypeLite.getDescription(), caseTypeEntity.getDescription());
+            assertEquals(caseTypeLite.getName(), caseTypeEntity.getName());
+            assertTrue(caseTypeLite.getStates().size() == 1);
```

Comment:
Replace with `assertEquals(1, jurisdiction.getCaseTypes().size());` for consistency elsewhere in the test, rather than using `assertTrue`.